samples/boards/nrf/ieee802154/802154_rpmsg/boards/nrf5340dk_nrf5340_cpunet.overlay

```diff
@@ -0,0 +1,35 @@
+/* Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
Is this a correct license for a file in the main Zephyr tree?

Reply:
> > > A second possible approach would be to pass low-power overlays during the creation of the child image here:
> > > https://github.com/nrfconnect/sdk-nrf/blob/06505e62904cbf049c8d1fa8ef04a0fb287d719c/samples/CMakeLists.txt#L62
> > 
> > 
> > This works only for configuration overlays not a DTS files.
> 
> But if this is needed, we should add support for it.
> 
> > As mentioned we have already discussed if we should implement a mechanism for passing DTS overlays to child images, but the conclusion was that in this case all unneeded peripherals should be disabled by default.
> 
> but why does this has to be in the `prj.conf` ?
> Why not make a `prj_low_power.conf` or `prj_power_optimized.conf` for this purpose, according to:
> https://docs.zephyrproject.org/latest/guides/kconfig/setting.html#the-initial-configuration
> 
> Will require a small addition to DTS overlay so that `nrf5340dk_nrf5340_cpunet_low_power.overlay` is used if present, but that should actually be made anyway.
> This PR just made me realize this was not done as part of fixing this:
> #27934

Thanks for the draft, we will need it as some point.
`prj.conf` case we discussed F2F should be addressed now.